source/docs/03-metadata.xml

```diff
@@ -1677,6 +1677,53 @@
+                        <specDesc key="exhibHist"/>
+                     </specList>
+                  </p>
+                  <p>The <gi scheme="MEI">exhibHist</gi> element is intended to contain descriptions of one or more public exhibitions of a bibliographic item. Often exhibitions include an additional description in the form of a tag for the public that accompanies the item on display. These descriptions may even be printed in a published exhibition catalogue, so the encoding may also include information about why the object was shown or what was significant about the exhibition. When formulating the encoding, it is at the discretion of the encoder whether to opt for text or more structured information. Text-centred encoding is made possible by <gi scheme="MEI">p</gi> elements in <gi scheme="MEI">exhibHist</gi>, among others. For more structured encodings, it is recommended to use the <gi scheme="MEI">eventList</gi> element contained in <gi scheme="MEI">exhibHist</gi>. Within MEI <gi scheme="MEI">exhibHist</gi> is conceptually bound to the item-level. As an element it is thus not permitted at the work or expression level and only permitted at the manifestation level, if the manifestation comprises a manifestation singleton.</p>
+                  <p>
+                     <specList>
+                        <specDesc key="physLoc"/>
+                     </specList>
+                  </p>
+                  <p>The <gi scheme="MEI">physLoc</gi> element allows encoding of any information related to or associated with the physical location of a bibliographic item. This includes, but is not limited to, the name of the holding institution, name or number of the building or room, any shelfmarks, etc. for the purpose of retrieval. The level of detail or machine readability of the encoding is generally at the discretion of the encoder and may vary depending on the information available. The <gi scheme="MEI">physLoc</gi> element may contain the following MEI elements:</p>
+                  <p>
+                     <specList>
+                        <specDesc key="head"/>
+                        <specDesc key="repository"/>
+                        <specDesc key="identifier"/>
+                        <specDesc key="history"/>
+                     </specList>
+                  </p>
+                  <p>The <gi scheme="MEI">repository</gi> element is intended to contain a description of the institution or individual currently holding the bibliographic item. Its content is either prose or structured markup. The <gi scheme="MEI">history</gi> element, on the other hand, is a container for additional non-bibliographic details regarding the physical location of an item. It may contain the elements <gi scheme="MEI">acquisition</gi>, <gi scheme="MEI">exhibHist</gi> or <gi scheme="MEI">provenance</gi>, among others,  to describe any events that coincided with a change of location, such as exhibitions, or change of custody.</p>
```

Comment:
```suggestion
                  <p>The <gi scheme="MEI">repository</gi> element is contains a description of the institution or individual currently holding the bibliographic item. Its content is either prose or structured markup. The <gi scheme="MEI">history</gi> element, on the other hand, is a container for additional non-bibliographic details regarding the physical location of an item. It may contain the elements <gi scheme="MEI">acquisition</gi>, <gi scheme="MEI">exhibHist</gi> or <gi scheme="MEI">provenance</gi>, among others,  to describe any events that coincided with a change of location, such as exhibitions, or change of custody.</p>
```